google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -726,11 +726,9 @@ public ReadChannel reader(BlobSourceOption... options) {
    * <pre>{@code
    * byte[] content = "Hello, World!".getBytes(UTF_8);
    * try (WriteChannel writer = blob.writer()) {
-   *   try {
    *     writer.write(ByteBuffer.wrap(content, 0, content.length));
-   *   } catch (Exception ex) {
-   *     // handle exception
-   *   }
+   * } catch (IOException ex) {
+   *   // handle exception
    * }
    * }</pre>
```

Comment:
you still have an extra closing brace

Reply:
@elharo ,I have gone throught code and found there is no extra closing brace.Here last closing 
brace indicate closing of {@code } tag.Can you please look once?

Reply:
line 733, just before </pre>, same below

Reply:
@elharo 
```
   * <pre>{@code
   * byte[] content = "Hello, World!".getBytes(UTF_8);
   * try (WriteChannel writer = blob.writer()) {
   *     writer.write(ByteBuffer.wrap(content, 0, content.length));
   * } catch (IOException ex) {
   *   // handle exception
   * }
   * }</pre>
```
Here First closing ```}``` brace indicate closing of try block,Second } brace indicate closing 
for catch clause.and last one ```}``` indicate closing of ```<pre>{@code``` so can we write document comment without ending ```<pre>{@code```?

Reply:
By my count you have two open braces and three close braces. Change line 733 from

`}</pre>` to `</pre>`

Reply:
Done